solr/test-framework/src/java/org/apache/solr/util/SolrClientTestRule.java

```diff
@@ -68,7 +68,7 @@ public NewCollectionBuilder(String name) {
       this.name = name;
     }
 
-    public NewCollectionBuilder withConfigSet(String configSet) {
+    public NewCollectionBuilder withConfigSet(Path configSet) {
```

Comment:
I appreciate this change, but I feel this should be overloaded so a configSet can be referenced by name instead of path.  For that, we can just trivially set the field (one liner).